vaccine_feed_ingest/runners/ri/arcgis/normalize.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python
+
+import datetime
+import json
+import logging
+import os
+import pathlib
+import re
+import sys
+from typing import List, Optional
+
+# import schema
+site_dir = pathlib.Path(__file__).parent
+state_dir = site_dir.parent
+runner_dir = state_dir.parent
+root_dir = runner_dir.parent
+sys.path.append(str(root_dir))
+
+from schema import schema  # noqa: E402
+
+# Configure logger
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s %(levelname)s:%(name)s:%(message)s",
+    datefmt="%m/%d/%Y %H:%M:%S",
+)
+logger = logging.getLogger("ak/arcgis/normalize.py")
+
+
+def _get_id(site: dict) -> str:
+    data_id = site["attributes"]["OBJECTID"]
+
+    # Could parse these from directory traversal, but do not for now to avoid
+    # accidental mutation.
+    site = "arcgis"
+    runner = "ri"
+
+    # Could parse these from the input file name, but do not for now to avoid
+    # accidental mutation.
+    arcgis = "da57e8c8663048a2a9893c636fef63d0"
+    layer = 0
+
+    return f"{runner}:{site}:{arcgis}_{layer}:{data_id}"
+
+
+def _get_inventory(site: dict) -> Optional[List[schema.Vaccine]]:
+    vaccines_field = site["attributes"]["USER_VaxType"].lower().split(",")
+
+    potentials = {
+        "pfizer-biontech covid-19 vaccine": schema.Vaccine(vaccine="pfizer"),
+        "moderna covid-19 vaccine": schema.Vaccine(vaccine="moderna"),
+        "janssen": schema.Vaccine(vaccine="janssen"),
+        "jjj": schema.Vaccine(vaccine="janssen"),
+        "varies": schema.Vaccine(vaccine="varies"),
+    }
+
+    inventory = []
+
+    for vf in vaccines_field:
+        try:
+            if vf != "-" and vf != "\x08":  # "-" is listed when no data given
+                inventory.append(potentials[vf])
+        except KeyError as e:
+            logger.error("Unexpected vaccine type: %s", e)
+
+    if len(inventory) > 0:
+        return inventory
+
+    return None
+
+
+def _get_contacts(site: dict) -> Optional[List[schema.Contact]]:
```

Comment:
It looks like in some cases this is adding an output phone number of `( ) -` to the dataset. Could you take a look at why that might be happening?

Reply:
Thanks for highlighting. Fixed.